sys/include/net/ng_pktbuf.h

```diff
@@ -57,33 +57,37 @@ extern "C" {
 #endif  /* NG_PKTBUF_SIZE */
 
 /**
+ * @brief   Initializes packet buffer module.
```

Comment:
Maybe you can extend this by something like: Resets the whole packet buffer when called "in the middle of a program" ?